src/additional-components/MiniMap/index.tsx

```diff
@@ -12,6 +12,7 @@ type StringFunc = (node: Node) => string;
 
 interface MiniMapProps extends React.HTMLAttributes<SVGSVGElement> {
   nodeColor?: string | StringFunc;
+  nodeClassName?: string | StringFunc;
```

Comment:
Do we really need the option to pass a function here or would it be enough to pass a string?

Reply:
We need a different class based on the node type. This can only be achieved with a StringFunc isn't ?

Reply:
Yep, that's correct. Then it's fine.